alphagov/paas-cf_basic_auth_route_service
# Simplify deployment

What
---

Route services are kinda complicated, involving a couple of moving parts. We could make this a bit easier for users of this route service by automating some of that.

How to review
---
1. Code review
2. Deploy an app you want to protect with the basic auth route service
3. Run the deploy script, targeting the app you want to protect.
4. Check it runs successfully
5. See if your app has been protected
